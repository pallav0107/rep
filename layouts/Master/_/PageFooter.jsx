import React from 'react';
import { Footer } from '@wdpui/react-gel';
import { Text } from '@wdpui/gel-typography';

export const PageFooter = props => (
  <Footer>
    <Text>
      Our site and your transactions are secure. You can read our{' '}
      <a href="">security information</a>.
      <br />
      test application
    </Text>
  </Footer>
);

export default PageFooter;
