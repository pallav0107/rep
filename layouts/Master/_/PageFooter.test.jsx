import React from 'react';
import { shallow } from 'enzyme';
import PageFooter from './PageFooter';
import { withTheme } from '../../../../utils/jest/TestUtils';
import "../../../setupTests"

describe('PageFooter', () => {
  it('should render the page footer', () => {
    const component = withTheme(<PageFooter />);
    const wrapper = shallow(component);
    expect(wrapper).toMatchSnapshot();
  });
});
