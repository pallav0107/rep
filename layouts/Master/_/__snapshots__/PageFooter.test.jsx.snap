// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter should render the page footer 1`] = `<Component />`;
